test/integration/create-next-app/index.test.ts

```diff
@@ -395,4 +395,52 @@ describe('create next app', () => {
       )
     })
   })
+
+  it('should use pnpm as the package manager on supplying --use-pnpm', async () => {
+    await usingTempDir(async (cwd) => {
+      const projectName = 'use-npm'
```

Comment:
```suggestion
      const projectName = 'use-pnpm'
```

Should this be pnpm?